core-plugins/src/test/java/co/cask/hydrator/plugin/transform/PythonEvaluatorTest.java

```diff
@@ -204,12 +204,37 @@ public void testSchemaValidation() throws Exception {
                                                                    ImmutableMap.<String, Object>of(
                                                                      CoreValidator.ID, new CoreValidator()));
     new PythonEvaluator(config).configurePipeline(configurer);
+  }
+
+  @Test
+  public void testSchemaValidation() throws Exception {
+    Schema outputSchema = Schema.recordOf(
+      "smallerSchema",
+      Schema.Field.of("intField", Schema.of(Schema.Type.INT)),
+      Schema.Field.of("longField", Schema.of(Schema.Type.LONG)));
+
+    PythonEvaluator.Config config = new PythonEvaluator.Config(
+      "def transform(x, emitter, context):\n" +
+        " y = {}\n" +
+        " y['intField'] *= 1024\n" +
+        " y['longField'] *= 1024\n" +
+        " emitter.emit(y)",
+      outputSchema.toString());
```

Comment:
Sorry, my bad. I just looked at the name of the test case. It's called `testScriptCompilationValidation` and it expects an `Exception` to be thrown. As a result, I think this "syntax error" is expected.